src/components-examples/material/table/table-row-binding/table-row-binding-example.ts

```diff
@@ -0,0 +1,50 @@
+import {Component} from '@angular/core';
+
+export interface PeriodicElement {
+  name: string;
+  position: number;
+  weight: number;
+  symbol: string;
+}
+
+const ELEMENT_DATA: PeriodicElement[] = [
+  {position: 1, name: 'Hydrogen', weight: 1.0079, symbol: 'H'},
+  {position: 2, name: 'Helium', weight: 4.0026, symbol: 'He'},
+  {position: 3, name: 'Lithium', weight: 6.941, symbol: 'Li'},
+  {position: 4, name: 'Beryllium', weight: 9.0122, symbol: 'Be'},
+  {position: 5, name: 'Boron', weight: 10.811, symbol: 'B'},
+  {position: 6, name: 'Carbon', weight: 12.0107, symbol: 'C'},
+  {position: 7, name: 'Nitrogen', weight: 14.0067, symbol: 'N'},
+  {position: 8, name: 'Oxygen', weight: 15.9994, symbol: 'O'},
+  {position: 9, name: 'Fluorine', weight: 18.9984, symbol: 'F'},
+  {position: 10, name: 'Neon', weight: 20.1797, symbol: 'Ne'},
+];
+
+/**
+ * @title Binding event handlers and properties to the table rows.
+ */
+@Component({
+  selector: 'table-row-binding-example',
+  styleUrls: ['table-row-binding-example.css'],
+  templateUrl: 'table-row-binding-example.html',
+})
+export class TableRowBindingExample {
+  displayedColumns: string[] = ['position', 'name', 'weight', 'symbol'];
+  dataSource = ELEMENT_DATA;
+
+  clickedRows: PeriodicElement[] = [];
+
+  recordClick(rowData: PeriodicElement) {
+    this.clickedRows.push(rowData);
+  }
+
+  isClicked(rowData: PeriodicElement) {
+    for (const clickedRow of this.clickedRows) {
```

Comment:
I think that you can reduce this to `this.clickedRows.includes(rowData)`.

Reply:
True - I'm still in "support IE" mode but I suppose that time has passed

Reply:
I think that we have a polyfill for it. Otherwise you can use `indexOf() > -1` which is supported in IE.